### ISSUE TITLE
Can't expand facts for a country

### ISSUE THREAD

**Issue by pyrog** [vmarc/knooppuntnet#425]
After a click to a fact, the display is empty and the console display errors:
```
NullInjectorError: R3InjectorError(Standalone[e])[t -> t -> r -> r -> r]: 
  NullInjectorError: No provider for r!
```
Tested on Chrome, Firefox and Safari.

[Example](https://knooppuntnet.nl/en/analysis/hiking/fr/facts)
<img width="972" alt="Capture d’écran 2024-06-04 à 06 43 19" src="https://github.com/vmarc/knooppuntnet/assets/2158081/8fc89368-033b-431a-b5ee-0952b8653498">

**Comment by pyrog** [comment]
Work fine for:
- a network (i.e. [Grand Chambéry](https://knooppuntnet.nl/en/analysis/network/13948824/facts))
- a locations (even a country: [France](https://knooppuntnet.nl/en/analysis/hiking/fr/France/facts))

**Comment by vmarc** [comment]
Fixed in v4.3.52.
